source/guides/infrastructure/admin-tools/ruby/mac.html.md.erb

```diff
@@ -0,0 +1,29 @@
+---
+title: MAC Installation  
+summary: Mac Setup (as of Catalina 11-07-20)
+weight: 5
+last_reviewed_on: 2021-11-30
+review_in: 1 year
+layout: navigation
+---
+
+1. install homebrew, this will ask to install the xcode command line tools if you dont have it already
+Run `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"`
+2. install gpg2
+Run `brew install gpg2`
+3. import the RVM keys *do not use these keys* get them from the official website (https://rvm.io/rvm/security) as they change often
```

Comment:
We generally use `asdf` in BAT — worth putting into the guidance here in favour of RVM (or just say "use your ruby version manager, ymmv 😛 "?)

Reply:
Shall we add this later? I added a comment to https://trello.com/c/puUi9mUo